src/main/java/io/github/christophermanahan/carnitas/Main.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         try (ServerSocket serverSocket = new ServerSocket(port(args))) {
             new HTTPServer(
               new RequestParser(),
-              new RequestHandler(),
+              Application.router(),
```

Comment:
I don't love the API of the `Application`, but I suppose it is fine for now. I would prefer to see something that looks like.
```suggestion
              new Application(),
```

Reply:
I also agree that this would be an improvement, I will keep this in mind.

Reply:
API improved!